news/12350-conda-forge-ci

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* <news item>
+
+### Other
+
+* Add support for conda-forge-based CI runtimes. (#12350)
```

Comment:
Let's be more verbose what that means, which version of Python do we test with conda-forge-built packages?